librocksdb_sys/crocksdb/c.cc

```diff
@@ -213,7 +213,16 @@ struct crocksdb_seqfile_t         { SequentialFile*   rep; };
 struct crocksdb_randomfile_t      { RandomAccessFile* rep; };
 struct crocksdb_writablefile_t    { WritableFile*     rep; };
 struct crocksdb_filelock_t        { FileLock*         rep; };
-struct crocksdb_logger_t          { shared_ptr<Logger>  rep; };
+struct crocksdb_logger_t : public Logger{
```

Comment:
Let's keep `crocksdb_logger_t` as is, and implement a separate `crocksdb_logger_impl_t` that implements `Logger`. That way existing `crocksdb_create_log_from_options` will still work.

Reply:
Also do me a favor, rename `crocksdb_create_log_from_options` to `crocksdb_create_logger_from_options`.

Reply:
Done